setup.cfg

```diff
@@ -0,0 +1,34 @@
+[metadata]
+name = cibyl
```

Comment:
Note for now it was `name='Cibyl'` in `setup.py`, so after installing, in `pip freeze` the name will be changed from `Cibyl` to `cibyl`. I am not sure if we care, but I just want to put your attention on that.